src/thorlabs_cube/driver/kcube/kpz.py

```diff
@@ -0,0 +1,193 @@
+import struct as st
+from typing import Tuple
```

Comment:
```suggestion
```
`tuple` type no longer needs to be imported from the `typing` library as of Python 3.9.
https://docs.python.org/3/library/typing.html#annotating-tuples